AutoFinance.Broker.IntegrationTests/InteractiveBrokers/Wrappers/EClientSocketTests.cs

```diff
@@ -753,6 +753,60 @@ public void ReqHistoricalData_Should_Callback()
             mockTwsWrapper.Verify(mock => mock.historicalData(requestId, It.IsAny<Bar>()));
             mockTwsWrapper.Verify(mock => mock.historicalDataEnd(requestId, It.IsAny<string>(), It.IsAny<string>()));
         }
+        
```

Comment:
The tests in this class mostly help to understand how TWS works, rather than actually testing the code of the library. The integration tests which are in the TwsControllerBaseTests and TwsControllerTests classes are more helpful, and should also be a bit easier to write. A test covering this feature would look almost exactly like `TwsControllerBaseTests:HistoricalDataController_Should_RetrieveHistoricalData`, but would listen for a `HistoricalDataUpdateEvent` and fail if one never comes through.